entity-types/ext-dbt_failed_resource_run/definition.yml

```diff
@@ -0,0 +1,67 @@
+domain: EXT
+type: DBT_FAILED_RESOURCE_RUN
+
+synthesis:
+  rules:
+    - identifier: entity_id 
+      name: entity_name 
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: dbt_resource_run 
+        - attribute: entity_name
+          present: True
+        - attribute: dbt_source
+          present: True
+        - attribute: status
+          value: fail 
+    - identifier: entity_id 
+      name: entity_name 
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: dbt_resource_run 
+        - attribute: entity_name
+          present: True
+        - attribute: dbt_source
+          present: True
+        - attribute: status 
+          value: warn 
+    - identifier: entity_id 
+      name: entity_name 
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: dbt_resource_run 
+        - attribute: entity_name
+          present: True
+        - attribute: dbt_source
+          present: True
+        - attribute: status 
+          value: error
```

Comment:
are there other statuses that we don't want to generate the entity from?

Wondering if we should create 1 rule with present:true instead.

Reply:
Just these statuses. This filters the number of entities created from 100k/day to just a few. The ones that fail are the only ones we need an entity for.